RELEASE-NOTES.txt

```diff
@@ -1,7 +1,10 @@
+2.10
+-----
```

Comment:
Interestingly, this file was updated to use 3.0 (as per "WordPress versioning") but the next milestone was created as 2.10 🤔 

2.10 is the version I'd expect, to be be consistent with what the other Simplenote apps (albeit inconsistent with the other apps in the division 😅).